### PR TITLE
Prepare release v0.80.0

### PR DESCRIPTION
**Pull request by jinja2**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,67 @@
...
+  The following replacement functions now accept a path expression as a replacement:
+  - replace_match
+  - replace_pattern
+  - replace_all_matches
+  - replace_all_patterns
+- (Contrib) `sapmexporter`: sapm exporter now supports `compression` config option to specify either gzip or zstd compression to use. ([#23257](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23257))
+- (Contrib) `sapmreceiver`: sapm receiver now accepts requests in compressed with zstd. ([#23257](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23257))
+- (Contrib) `exporter/signalfx`: Do not drop container.cpu.time metric in the default translations so it can be enabled in the include_metrics config. ([#23403](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23403))
+- (Contrib) `sqlqueryreceiver`: Add support for logs ([#20284](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/20284))
+- (Contrib) `k8sclusterreceiver`: Switch k8s.deployment metrics to use pdata. ([#23416](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23416))
```
Comment (jinja2) [inline CHANGELOG.md:37]: maybe I should make a single entry for these pdata adoption changes

Comment (atoulme) [reply CHANGELOG.md:37]: It's fine by me as it is

**Review by atoulme — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,67 @@
...
+- (Core) `service/pipelines`: Add pipelines.Config to remove duplicate of the pipelines configuration ([#7854](https://github.com/open-telemetry/opentelemetry-collector/issues/7854))
+
+### 🧰 Bug fixes 🧰
+
+- (Contrib) `otel-collector`: Fix cri-o log format time layout ([#23027](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23027))
+- (Contrib) `receiver/hostmetricsreceiver`: Fix not sending `process.cpu.utilization` when `process.cpu.time` is disabled. ([#23450](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23450))
+- (Contrib) `receiver/kafkametricsreceiver`: Updates certain metrics in kafkametricsreceiver to function as non-monotonic sums. ([#4327](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/4327))
+  Update the metric type in KafkaMetricsReceiver from "gauge" to "nonmonotonic sum".
+- (Contrib) `processor/hostmetrics`: Fix issue where receiver fails to read parent-process information for some processes on Windows ([#14679](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14679))
```
Comment (rmfitzpatrick) [inline CHANGELOG.md:62]:
```suggestion
- (Contrib) `receiver/hostmetrics`: Fix issue where receiver fails to read parent-process information for some processes on Windows ([#14679](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14679))
```

File: CHANGELOG.md (hunk 1 above)
Comment (rmfitzpatrick) [reply CHANGELOG.md:37]: Since pods (and maybe others?) are still in the works it may be ok as is. Defer to @atoulme though (🚀)

**Review by rmfitzpatrick — APPROVED** [review]

**Pull request merged**